showcase/app/templates/components/form/index.hbs

```diff
@@ -0,0 +1,1078 @@
+            <Shw::Placeholder @height="2.4em" @text="FormField" @background="#add8e6" />
+            <Shw::Placeholder @height="2.4em" @text="FormField" @background="#add8e6" />
+          </FORM.Section>
+        </Hds::Form>
+      </SG.Item>
+
+      <SG.Item @label="hasBorder=true">
+        <Hds::Form as |FORM|>
+          <FORM.Section @hasBorder={{true}}>
+            <Shw::Placeholder @height="2.4em" @text="FormField" @background="#add8e6" />
+            <Shw::Placeholder @height="2.4em" @text="FormField" @background="#add8e6" />
+            <Shw::Placeholder @height="2.4em" @text="FormField" @background="#add8e6" />
+          </FORM.Section>
+        </Hds::Form>
+      </SG.Item>
+
+      <SG.Item @label="Bordered Section nested in another bordered Section">
+        <Hds::Form as |FORM|>
+          <FORM.Section @hasBorder={{true}}>
+            <Shw::Placeholder @height="2.4em" @text="FormField" @background="#add8e6" />
+            <Shw::Placeholder @height="2.4em" @text="FormField" @background="#add8e6" />
+            <Shw::Placeholder @height="2.4em" @text="FormField" @background="#add8e6" />
+
+            <FORM.Section @hasBorder={{true}}>
+              <Shw::Placeholder @height="2.4em" @text="FormField" @background="#add8e6" />
+              <Shw::Placeholder @height="2.4em" @text="FormField" @background="#add8e6" />
+              <Shw::Placeholder @height="2.4em" @text="FormField" @background="#add8e6" />
+            </FORM.Section>
+
+            <Shw::Placeholder @height="2.4em" @text="FormField" @background="#add8e6" />
+          </FORM.Section>
+        </Hds::Form>
+      </SG.Item>
+    </Shw::Grid>
+  </div>
+
+  <Shw::Divider @level={{2}} />
+
+  <Shw::Text::H3>SectionHeader, SectionHeaderTitle, & Description</Shw::Text::H3>
+
+  <button type="button" class="shw-component-form-layout-button-highlight" {{on "click" this.toggleHighlight}}>
+    {{if this.showHighlight "Hide layout highlight" "Show layout highlight"}}
+  </button>
+
+  <div class="{{if this.showHighlight 'shw-component-form-layout-highlight-elements'}}">
+    <Hds::Form as |FORM|>
+      <FORM.Section>
+        <FORM.SectionHeader>
+          <FORM.SectionHeaderTitle>Section header title</FORM.SectionHeaderTitle>
+          <FORM.SectionHeaderDescription>
+            Section Header description—lorem ipsum dolor sit amet consectetur adipisicing elit. Quasi eum neque totam
+            vel facere itaque necessitatibus quisquam omnis deserunt! Dicta, explicabo quia quis odio nesciunt quibusdam
+            dolor obcaecati consequatur tenetur.
+          </FORM.SectionHeaderDescription>
+        </FORM.SectionHeader>
+      </FORM.Section>
+    </Hds::Form>
+  </div>
+
+  <Shw::Divider @level={{2}} />
+
+  <Shw::Text::H4>SectionHeaderTitle sizes</Shw::Text::H4>
+
+  <Shw::Grid @columns={{1}} @gap="1.4rem" as |SG|>
+    {{#each this.model.DISPLAY_AVAILABLE_SIZES as |size|}}
+      <SG.Item @label="size={{size}}">
```

Comment:
[suggestion] would be great to indicate when it's default using the exported `DEFAULT_SIZE` from `packages/components/src/components/hds/form/header/title/index.ts` and compare it with the `size` of the loop. something like this:
```suggestion
      <SG.Item @label="size={{size}}{{if (eq size SECTION_HEADER_TITLE_DEFAULT_SIZE) ' (default)'}}">
```

(same for the header title)

Reply:
I implemented this locally, thanks.